client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
What do we need to transpile? If the brower supports service workers, it's most likely than not supports ES6.